manager/dispatcher/dispatcher.go

```diff
@@ -572,20 +572,37 @@ func (d *Dispatcher) Tasks(r *api.TasksRequest, stream api.Dispatcher_TasksServe
 			return err
 		}
 
-		select {
-		case event := <-nodeTasks:
-			switch v := event.(type) {
-			case state.EventCreateTask:
-				tasksMap[v.Task.ID] = v.Task
-			case state.EventUpdateTask:
-				tasksMap[v.Task.ID] = v.Task
-			case state.EventDeleteTask:
-				delete(tasksMap, v.Task.ID)
+		// bursty events should be processed in batches and sent out snapshot
+		const eventBatchLimit = 200
```

Comment:
`eventBatchLimit = 200` is selected to send snapshots out at most every 10 seconds (10 seconds / 50 ms = 200). It was set to this relatively large value because in current implementation status updates from agent trigger events. I agree it should be reduce to start parallelism. How about 20 when we removes the agent status update events? 

A side note though, I think `start parallelism` should be handled by agent. Dispatcher just tells what to do, where agent decides how to do it.